### ISSUE TITLE
Export `alpha` argument in `element_rect()`

### ISSUE THREAD

**Issue by Yunuuuu** [tidyverse/ggplot2#6260]
I would like to request the addition of an `alpha` argument to the `element_rect()` function in ggplot2. Adding an alpha argument to element_rect() would make it easier for users to adjust the transparency of rectangular elements without needing to manually calling `ggplot2::alpha()` or `ggplot2::fill_alpha()`

**Comment by teunbrand** [comment]
For some more context: https://github.com/tidyverse/ggplot2/pull/6249#issuecomment-2551992220

I don't have any strong opinion on whether this should be exposed, but if we do it for `element_rect()`, we should do it for all elements.

**Comment by Yunuuuu** [comment]
Or we can add some documents, I found many user didn't know the `alpha` and `fill_alpha` function

**Comment by teunbrand** [comment]
Yeah let's go with just documenting how to set alpha on a colour. Alpha is separate from the colour/fill in geoms because they can be mapped from the data and thus be independent of a colour, whereas this generally doesn't occur with theme elements. Moreover, it would create some burden on extension developers to follow the same pattern in e.g. `ggtext::element_markdown()`, `marquee::element_marquee()`, `ggh4x::element_part_rect()` etc.
